script/github-actions/run-cypress-tests.js

```diff
@@ -26,16 +27,21 @@ const batch = tests
   .slice(step * divider, (step + 1) * divider)
   .join(',');
 
-if (longestTestIsPresent && step === lastStep) {
-  const status = runCommandSync(
-    `CYPRESS_EVERY_NTH_FRAME=1 yarn cy:run --browser chrome --headless --reporter cypress-multi-reporters --reporter-options "configFile=config/cypress-reporters.json" --spec 'src/applications/**/all-claims.cypress.spec.js' --env app_url=${appUrl}`,
-  );
-  process.exit(status);
-} else if (batch !== '') {
-  const status = runCommandSync(
-    `CYPRESS_EVERY_NTH_FRAME=1 yarn cy:run --browser chrome --headless --reporter cypress-multi-reporters --reporter-options "configFile=config/cypress-reporters.json" --spec '${batch}' --env app_url=${appUrl}`,
-  );
-  process.exit(status);
-} else {
-  process.exit(0);
+let status = null;
+const upTo = isStressTest ? 25 : 1;
```

Comment:
Fair comment. Maybe it's because my first language was Ruby and there's an [Integer#upto](https://ruby-doc.org/core-2.5.0/Integer.html#method-i-upto) method. I renamed this var `runTestInLoopUpTo`. Does that work for you?

Reply:
Ohh yeah, its been a while since I've touched ruby, but I remember the `upTo` function. I think the name you suggested sounds good. Since this often runs multiple tests in each loop, it may be worth adding an `s` after `Tests`, `runTestsInLoopUpTo`

Reply:
Cool, just made it plural.